src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -1202,4 +1212,38 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
     ENDIF.
 
   ENDMETHOD.
+
+
+  METHOD belongs_incl_to_other_fugr.
+    " make sure that the include belongs to the function group
+    " like in LSEAPFAP Form TADIR_MAINTENANCE
+    DATA ls_tadir TYPE tadir.
+    DATA lv_namespace TYPE rs38l-namespace.
+    DATA lv_area TYPE rs38l-area.
+    DATA lv_include TYPE rs38l-include.
+
+    rv_belongs_to_other_fugr = abap_false.
+    IF iv_include(1) = 'L' OR iv_include+1 CS '/L'.
```

Comment:
Namespace split/check is not correct. Includes are /.../L...

Reply:
@mbtools Hi Marc, why do think it is not ok? I implemented like the SAP Standard. iv_include+1 CS '/L' means search  '/L' after the first '/' (iv_include+0).

![grafik](https://user-images.githubusercontent.com/59128840/141697508-ff2ecebb-a6da-4074-b62e-520a4760ebe1.png)

Reply:
It is not possible to have more than 2 / in a program name.
![grafik](https://user-images.githubusercontent.com/59128840/141730217-a53d8e28-e90f-4391-b07d-1d9192766037.png)

It is also not possible to have 2 / and the first one is not the first character.
![grafik](https://user-images.githubusercontent.com/59128840/141730522-897ec573-2c98-4603-8638-fd7482da5f18.png)

Reply:
that's not it. in your code, `lv_namespace` isn't set. (sorry, my marker wasn't precise. )

Reply:
you are right (i read it as `=`)